app/utils/aws.ts

```
@@ -0,0 +1,432 @@
+
+  return headers;
+}
+
+export function extractMessage(res: any, modelId: string = ""): string {
+  if (!res) {
+    console.error("[AWS Extract Error] extractMessage Empty response");
+    return "";
+  }
+  console.log("[Response] extractMessage response: ", res);
+  return res?.content?.[0]?.text;
+  return "";
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)